Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
```

Comment:
Need to try and figure out a way to not set/get instance variables here. If need be: delete the relevant tests that need them set given we're almost at the end of GSoC. Ideally: just call other methods that end up calling these ones.